### PR TITLE
Rebuild openstack/features page

### PR DESCRIPTION
**Pull request by bethcollins92**
## Done

-  Rebuild `openstack/features` 

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/openstack/features
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check the page against the [copy doc](https://docs.google.com/document/d/1odFXuFtN1GjuZvgd8tW48HB4J9PjZJtP3w6wY0xlmog/edit#) 

## Issue / Card

Fixes [#3872](https://github.com/canonical-web-and-design/web-squad/issues/3872)

### CODE REVIEW

**Comment by anasereijo** [comment]
few changes and it's good to go:

- [ ] make text 6 cols

Screenshot 2021-03-12 at 10 26 43

- [ ] @SirSamTumless when will this diagram be going to be ready? This doesn't look right
Screenshot 2021-03-12 at 10 29 13

**Comment by bethcollins92** [comment]
@anasereijo replaced the diagram - the background of the new image is white so I changed the strip colour back to white. Ready for re-review, thanks

File: templates/openstack/features.html [hunk 1]
```diff
@@ -88,321 +291,425 @@ <h2>VMware migration</h2>
...
-        <li class="p-list__item is-ticked">Daily operations are fully automated</li>
-        <li class="p-list__item is-ticked">Upgrades are guaranteed for every OpenStack version</li>
-        <li class="p-list__item is-ticked">Security improves for every deployment at the same time</li>
-        <li class="p-list__item is-ticked">Third-party integrations are shared as soon as they are ready</li>
-      </ul>
-      <p>You don't need to hire a large team that follows every aspect of OpenStack development. Just focus on high-level operations and your business workloads.</p>
-      <p><a href="https://pages.ubuntu.com/stuckstack_webinar_upgrades_register.html" class="p-link--external">Watch our OpenStack Upgrade webinar</a></p>
+  <div class="row u-vertically-center">
+    <div class="col-6">
+      <h3>OpenStack operators package cloud operations</h3>
```
Comment (albertkol) [inline templates/openstack/features.html:306]: I think this should say "OpenStack Charmed Operators..."

File: templates/openstack/features.html [hunk 2]
```diff
@@ -88,321 +291,425 @@ <h2>VMware migration</h2>
...
+        ) | safe
+      }}
     </div>
   </div>
-  <div class="row">
-    <div class="col-9">
-      <h4 class="p-heading--five">MAAS enables multi-vendor and multi-OS data centre operations:</h4>
+  <div class="row u-equal-height u-vertically-center">
+    <div class="col-12">
+      <h4>With <a href="https://charmhub.io/?q=operators" class="p-link--external">OpenStack operators (charms)</a>:</h4>
```
Comment (albertkol) [inline templates/openstack/features.html:323]: The text should say "OpenStack Charmed Operators (OpenStack Charms)"

File: templates/openstack/features.html [hunk 3]
```diff
@@ -2,79 +2,282 @@
 
 {% block title %}Features | OpenStack{% endblock %}
```
Comment (albertkol) [inline templates/openstack/features.html:3]: Should the title not be: "Carrier-grade private cloud - Canonical’s Charmed OpenStack"?

**Review by albertkol — COMMENTED** [review]
Holy Molly - Good job, Beth!

3 small mentions.

**Comment by bethcollins92** [comment]
Thanks @albertkol - made those changes 👍🏼

**Review by albertkol — APPROVED** [review]
:+1:

**Pull request merged**